iis/moduleconfig.h

```diff
@@ -71,6 +71,8 @@ class MODSECURITY_STORED_CONTEXT : public IHttpStoredContext
 
     directory_config* config = nullptr;
 
+    BOOL configLoadingFailed = false;
```

Comment:
`BOOL` (uppercase) is a Windows-specific `typedef` for int, the built-in type name is `bool`.
So it is better to use `bool` with `true`/`false` values to stay consistent.

Reply:
Sure. All the other types in this struct are the windows typedefs, that's why I stuck with it for consistency.